zebra-rpc/src/methods/tests/snapshot/snapshots/get_block_subsidy_future_nu6_height@testnet_10.snap

```diff
@@ -1,28 +0,0 @@
----
-source: zebra-rpc/src/methods/tests/snapshot/get_block_template_rpcs.rs
-expression: get_block_subsidy
----
-{
-  "fundingstreams": [
-    {
-      "recipient": "Zcash Community Grants NU6",
-      "specification": "https://zips.z.cash/draft-nuttycom-funding-allocation#alternative-2-hybrid-deferred-dev-fund-transitioning-to-a-non-direct-funding-model",
-      "value": 0.125,
-      "valueZat": 12500000,
-      "address": "t2HifwjUj9uyxr9bknR8LFuQbc98c3vkXtu"
-    }
-  ],
-  "lockboxstreams": [
-    {
-      "recipient": "Lockbox NU6",
-      "specification": "https://zips.z.cash/draft-nuttycom-funding-allocation#alternative-2-hybrid-deferred-dev-fund-transitioning-to-a-non-direct-funding-model",
-      "value": 0.1875,
-      "valueZat": 18750000
-    }
-  ],
-  "miner": 1.25,
-  "founders": 0.0,
-  "fundingstreamstotal": 0.125,
-  "lockboxtotal": 0.1875,
-  "totalblocksubsidy": 1.5625
-}
```

Comment:
Is it possible to run the tests to get this output ?

Reply:
Yep. I made a mistake in the refactor.

Reply:
Yep, I chose a NU6 activation height below of the funding stream height range start bound.